### ISSUE TITLE
v is  not defined when page in frameset

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#57]
```
What steps will reproduce the problem?
1. when a page use ie7.js in a frameset and all page use xhtml1.1 dtd
2. childpage promot "v is not  defined"
3.

What is the expected output? What do you see instead?
no error

What version of the product are you using? On what operating system?
ie6 windows

Please provide any additional information below.

error line:
 v=document.body;s=C?v:bm;v.className+=" ie7_body"
 i change to :
v=document.body?document.body:document.documentElement;


me:  lxbzmy in gmail
```

Original issue reported on code.google.com by `lxb...@gmail.com` on 19 Mar 2008 at 2:31

**Comment by GoogleCodeExporter** [comment]
```
I made some test and find reason :
final builded code can not use in frame,because 'document' is not frame's 
document
,document is frameset's 'document' ,so document.body is null;
the initialisation method should chang.
```

Original comment by `lxb...@gmail.com` on 22 Mar 2008 at 2:46

**Comment by GoogleCodeExporter** [comment]
```
I posted the solution I came up with:

http://groups.google.com/group/ie7-js/msg/160049c734113532
```

Original comment by `john.lov...@gmail.com` on 19 May 2009 at 8:15

**Comment by GoogleCodeExporter** [comment]
```
Can you provide a test page please?
```

Original comment by `dean.edw...@gmail.com` on 11 Feb 2010 at 1:19

**Comment by GoogleCodeExporter** [comment]
```
Issue 77 has been merged into this issue.
```

Original comment by `dean.edw...@gmail.com` on 11 Feb 2010 at 7:24

**Comment by GoogleCodeExporter** [comment]
```
Issue 91 has been merged into this issue.
```

Original comment by `dean.edw...@gmail.com` on 11 Feb 2010 at 7:24

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in version 2.1.
```

Original comment by `dean.edw...@gmail.com` on 11 Feb 2010 at 7:26
- Changed state: **Fixed**
